core/spec/lib/spree/core/importer/order_spec.rb

```diff
@@ -390,6 +390,32 @@ module Core
         expect(order.adjustments.first.amount).to eq -4.99
       end
 
+      it 'adds line item adjustments' do
+        params = {
+          line_items_attributes: line_items,
+          adjustments_attributes: [
+            { label: 'Line Item Discount', amount: -4.99, variant_id: line_items.first[:variant_id] },
+            { label: 'Order Discount', amount: -5.99 }
+          ]
+        }
+
+        order = Importer::Order.import(user, params)
+
+        line_item_adjustment = order.line_item_adjustments.first
+        expect(line_item_adjustment.closed?).to be true
```

Comment:
@BenMorganIO I believe be_truthy will pass for any positive return like say 1 would be truthy where as here you explicitly want the value to be true not just truthy. http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers:be_truthy